content/chainguard/chainguard-enforce/events/create-github-issues/index.md

```diff
@@ -0,0 +1,238 @@
+
+Run the following `chainctl` command to create an Enforce webhook subscription:
+
+```shell
+chainctl events subscriptions create \
+  --group <your chainguard group> \
+  https://<url from terraform>
+```
+
+You will receive output showing the ID of the subscription and the webhook endpoint, like the following:
+
+```
+                    ID                   |             SINK
+ ----------------------------------------+-----------------------------
+  <chainguard group ID>/0dfbc3c8b89e0147 | https://<url from terraform>
+  ```
+
+Enforce will now send events to the application that is running on Cloud Run at the specified URL.
+
+## Deploying a Pod That Will Violate a Policy
+
+In this step you will create a Kubernetes deployment that uses an unsigned image. Run the following to deploy `nginx` into your cluster:
+
+```shell
+kubectl create deployment nginx --image=nginx
+```
+
+Now that you have a workload running, you can can add an Enforce policy, and the deployment will trigger a policy violation event, which in turn will create a GitHub issue.
+
+## Configuring an Enforce Policy
+
+Before you can open GitHub issues for policy violations, you will need to ensure that you have Enforce configured with a policy that you can test. Make sure you have a cluster registered with Enforce using the `chainctl clusters ls` command. If you do not, visit our [Getting Started with Enforce Guide](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-user-onboarding/#step-3--prepare-kubernetes-cluster) to create and register a cluster.
+
+Once you have a cluster enrolled with Enforce, create the following policy from the Getting Started guide:
+
+```
+cat > sample-policy.yaml <<EOF
+apiVersion: policy.sigstore.dev/v1beta1
+kind: ClusterImagePolicy
+metadata:
+  name: sample-policy
+spec:
+  images:
+  - glob: "ghcr.io/chainguard-dev/*/*"
+  - glob: "ghcr.io/chainguard-dev/*"
+  - glob: "index.docker.io/*"
+  - glob: "index.docker.io/*/*"
+  - glob: "cgr.dev/chainguard/**"
+  authorities:
+  - keyless:
+      url: https://fulcio.sigstore.dev
+      identities:
+        - issuerRegExp: ".*"
+          subjectRegExp: ".*"
+EOF
+```
+
+Apply it using `chainctl`:
+
+```
+chainctl policies apply \
+  -f sample-policy.yaml --group=<your chainguard group>
+```
+
+Depending on your Kubernetes cluster, applying the policy may generate multiple GitHub issues. For example, if you are using `kind`, the control plane images are not signed and should create issues.
+
+Once the policy is created, check the logs of your [Cloud Run deployment](https://console.cloud.google.com/logs/query;query=resource.type %3D "cloud_run_revision"%0Aresource.labels.service_name %3D "enforce-events-issue-opener"%0AtextPayload: "Opened issue";timeRange=PT4H). You may need to edit the `service_name` portion of the URL if your service is called something different from `enforce-events-issue-opener`.
```

Comment:
Annoyingly, this doesn't work because of the URL encoding. Will figure out how to updated it before merging.